CMakeLists.txt

```diff
@@ -131,21 +131,44 @@ option( BUILD_SHARED_LIBS "Build rocBLAS as a shared library" ON )
 include( clients/cmake/build-options.cmake )
 
 if( BUILD_WITH_TENSILE )
-  include(virtualenv)
   set( Tensile_LOGIC "asm_full" CACHE STRING "Tensile to use which logic?")
-  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
 
   option( Tensile_MERGE_FILES "Tensile to merge kernels and solutions files?" ON )
   option( Tensile_SHORT_FILENAMES "Tensile to use short file names? Use if compiler complains they're too long." OFF )
   option( Tensile_PRINT_DEBUG "Tensile to print runtime debug info?" OFF )
 
-  set( tensile_tag "develop" CACHE STRING "Tensile tag to download" )
-  virtualenv_install("git+https://github.com/ROCmSoftwarePlatform/Tensile.git@${tensile_tag}")
-  list(APPEND CMAKE_PREFIX_PATH ${VIRTUALENV_HOME_DIR})
+  set( Tensile_TEST_LOCAL_PATH "" CACHE PATH "Use local Tensile directory instead of fetching a GitHub branch" )
+  set( CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories" )
+  set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite hip_lite other )
+
+
+  if (Tensile_TEST_LOCAL_PATH)
+    # Use user's local Tensile path:
+    execute_process( COMMAND ${CMAKE_COMMAND} -E copy_directory ${Tensile_TEST_LOCAL_PATH} ${PROJECT_EXTERN_DIR}/Tensile)
```

Comment:
This should install with `virtualenv_install("-e ${Tensile_TEST_LOCAL_PATH}")` so it will properly install the dependencies for tensile. Also, this links the source files back to tensile, so as you make changes it will be updated in rocblas as well.